x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/policy_template_form.test.tsx

```diff
@@ -231,47 +244,29 @@ describe('<CspPolicyTemplateForm />', () => {
       })
     );
 
-    (useParams as jest.Mock).mockReturnValue({
-      integration: 'kspm',
-    });
-    render(<WrappedComponent newPolicy={policy} />);
-
-    // 1st call happens on mount and selects the default policy template enabled input
-    expect(onChange).toHaveBeenNthCalledWith(1, {
-      isValid: true,
-      updatedPolicy: {
-        ...getMockPolicyK8s(),
-      },
-    });
-  });
-
-  it('selects default CSPM input selector', () => {
-    const policy = getMockPolicyAWS();
-    // enable all inputs of a policy template, same as fleet does
-    policy.inputs = policy.inputs.map((input) => ({
-      ...input,
-      enabled: input.policy_template === 'cspm',
-    }));
-
-    (useParams as jest.Mock).mockReturnValue({
-      integration: 'cspm',
-    });
-    (useCspSetupStatusApi as jest.Mock).mockImplementation(() =>
+    (usePackagePolicyList as jest.Mock).mockImplementation(() =>
       createReactQueryResponse({
         status: 'success',
         data: {
-          cspm: { status: 'not-deployed', healthyAgents: 0, installedPackagePolicies: 1 },
+          items: [
+            getPosturePolicy(getMockPolicyAWS(), CLOUDBEAT_AWS),
+            getPosturePolicy(getMockPolicyEKS(), CLOUDBEAT_EKS),
+            getPosturePolicy(getMockPolicyVulnMgmtAWS(), CLOUDBEAT_AWS),
+          ],
         },
       })
     );
 
-    render(<WrappedComponent newPolicy={policy} />);
+    const { debug } = render(
+      <WrappedComponent newPolicy={policy} packageInfo={{ name: 'kspm' } as PackageInfo} />
+    );
+    debug();
 
     // 1st call happens on mount and selects the default policy template enabled input
     expect(onChange).toHaveBeenNthCalledWith(1, {
       isValid: true,
       updatedPolicy: {
-        ...getMockPolicyAWS(),
+        ...getMockPolicyK8s(),
       },
     });
   });
```

Comment:
can you add a test to the policy_template_form where we expect the name to be kspm-2?

I expect to see something like this
```js
// ...
(useCspSetupStatusApi as jest.Mock).mockImplementation(() =>
	createReactQueryResponse({
        status: 'success',
        data: {
          items: [
						getPosturePolicy(getMockPolicyK8s(), CLOUDBEAT_VANILLA)
          ],
        },
      })
    );

// ...

expect(onChange).toHaveBeenNthCalledWith(1, {
      isValid: true,
      updatedPolicy: {
        ...getMockPolicyK8s(),
				name: 'kspm-2'
      },
    });
  });
```